extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -56,6 +59,10 @@ export class MethodModelingViewProvider extends AbstractWebviewViewProvider<
   ): Promise<void> {
     this.method = method;
     this.databaseItem = databaseItem;
+    this.language =
+      databaseItem && isQueryLanguage(databaseItem?.language)
```

Comment:
is this the same as `databaseItem && tryGetQueryLanguage(databaseItem?.language)`?